gluecodium/src/main/resources/templates/dart/DartInterface.mustache

```diff
@@ -146,12 +150,7 @@ class {{resolveName}}$Impl extends __lib.NativeBase implements {{resolveName}} {
   {{resolveName}}$Impl(Pointer<Void> handle) : super(handle);
 
   @override
-  void release() {
-    if (handle.address == 0) return;
-    __lib.uncacheInstance(handle);
-    _{{resolveName "Ffi"}}ReleaseHandle(handle);
-    handle = Pointer<Void>.fromAddress(0);
-  }
+  void release() {}
```

Comment:
Remove or deprecate?

Reply:
This is inside an `$Impl`, that implements the interface, so it must implement and override `release()`. Does not need to be deprecated though, since `$Impl` classes are not public.